Makefile

```
@@ -122,7 +122,7 @@ sagelib-clean:
 	     rm -rf c_lib .cython_version cython_debug; \
 	     rm -rf build; find . -name '*.pyc' -o -name "*.so" | xargs rm -f; \
 	     rm -f $$(find . -name "*.pyx" | sed 's/\(.*\)[.]pyx$$/\1.c \1.cpp/'); \
-	     rm -rf sage/ext/interpreters) \
+	     cd sage/ext/interpreters/; rm -f *.so *.c *.h *.py* *.pxd) \
```

```suggestion
	     cd sage/ext/interpreters/ && rm -f *.so *.c *.h *.py* *.pxd) \
```
I'm a bit wary of `rm -f` and wildcards. This seems dangerous if the directory was removed by some reason.

A milder alternative https://github.com/sagemath/sage/pull/36524#issuecomment-2461184829

---

done